app/db/knexfile.js

```diff
@@ -1,16 +1,12 @@
-let DATABASE_URL
+let DATABASE_URL = process.env.DATABASE_URL
 
-if (!process.env.NODE_ENV) {
-  process.env.NODE_ENV = 'development'
-}
-
-if (process.env.DSN) {
```

Comment:
@vgeorge Hydra takes a DSN environment variable, not DATABASE_URL, that's why the .env files use DSN

Reply:
@kamicut I revert the knexfile back the way it was before to reinstate DNS and updated the connection strings for the new docker db instances. Also updated the hydra connection string in docs (added port 5433). Ideally Hydra and Postgres should talk within a Docker network, but I think it should be ok the way it is as this setup is for development only. Please let me know if that is ok.

Reply:
@kamicut did an update to use Docker default network in the Hydra connection string, can you please have another look?